iothub/device/src/Transport/Amqp/AmqpAuthenticationRefresher.cs

```diff
@@ -181,7 +195,6 @@ private void Dispose(bool disposing)
                 {
                     if (disposing)
                     {
-                        StopLoop();
```

Comment:
Any token refresh attempts is now canceled from within the CloseAsync() call and not the Dispose() call.